test/ipc_os_prov.sh

```diff
@@ -20,19 +20,8 @@ VAL=0
 if [ -f $PTRACE_SCOPE_FILE ]; then
 	PTRACE_SCOPE_VAL=$(cat $PTRACE_SCOPE_FILE)
 	if [ $PTRACE_SCOPE_VAL -ne $VAL ]; then
```

Comment:
I think this test should NOT skip silently in CI, but it should always error out in CI:

```sh
	if [ $PTRACE_SCOPE_VAL -ne $VAL ]; then
		if [ "$CI" == "true" ]; then
			echo "ERROR: ptrace_scope is not set to 0 (classic ptrace permissions)"
			exit 1 # error out in case of CI
		else
			echo "SKIP: ptrace_scope is not set to 0 (classic ptrace permissions) - skipping the test"
			exit 125 # skip code defined in CMakeLists.txt
		fi
	fi
```

Reply:
Done.